pkg/controller/runtimeextractor/resources/daemonset.go

```
@@ -0,0 +1,144 @@
+	}
+
+	rm.updateContainerImages(daemonSet)
+
+	// ApplyDaemonSet handles create/update logic with generation tracking
+	appliedDaemonSet, modified, err := resourceapply.ApplyDaemonSet(ctx, rm.daemonSetGetterClient, rm.recorder, daemonSet, -1)
+	if err != nil {
+		return nil, fmt.Errorf("failed to apply runtime extractor daemonset: %w", err)
+	}
+
+	if modified {
+		rm.recorder.Event("DaemonSet Updated", fmt.Sprintf("Runtime extractor DaemonSet %s/%s was created or updated", appliedDaemonSet.Namespace, appliedDaemonSet.Name))
```

golangci-lint (2.11.4)

[error] The line is 164 characters long, which exceeds the maximum of 140 characters.

(lll)